client/my-sites/comments/comment-list/index.jsx

```diff
@@ -19,42 +19,34 @@ import {
 } from 'state/comments/actions';
 import { createNotice, removeNotice } from 'state/notices/actions';
 import { getNotices } from 'state/notices/selectors';
-import getSiteComments from 'state/selectors/get-site-comments';
 import CommentDetail from 'blocks/comment-detail';
 import CommentDetailPlaceholder from 'blocks/comment-detail/comment-detail-placeholder';
 import CommentNavigation from '../comment-navigation';
 import EmptyContent from 'components/empty-content';
 import Pagination from 'components/pagination';
-import QuerySiteComments from 'components/data/query-site-comments';
-import { hasSiteComments } from 'state/selectors';
+import QuerySiteCommentsTree from 'components/data/query-site-comments-tree';
+import { getSiteCommentsTree, hasSiteComments } from 'state/selectors';
 
 const COMMENTS_PER_PAGE = 2;
```

Comment:
Maybe we could bump this to 10? I feel like we might miss some cases with only 2 per page.